vm/src/vm/runners/builtin_runner/ec_op.rs

```diff
@@ -62,31 +60,30 @@ impl EcOpBuiltinRunner {
         partial_sum: (Felt252, Felt252),
         doubled_point: (Felt252, Felt252),
         m: &Felt252,
-        alpha: &BigInt,
-        prime: &BigInt,
         height: u32,
-    ) -> Result<(BigInt, BigInt), RunnerError> {
-        let mut slope = felt_to_bigint(*m);
-        let mut partial_sum_b = (felt_to_bigint(partial_sum.0), felt_to_bigint(partial_sum.1));
-        let mut doubled_point_b = (
-            felt_to_bigint(doubled_point.0),
-            felt_to_bigint(doubled_point.1),
-        );
+    ) -> Result<(Felt252, Felt252), RunnerError> {
+        let mut slope = m.to_bigint();
```

Comment:
Is the `slope` allowed to be negative? We're only considering the bits (not sign) in the body. So, I think that's either buggy or assumed positive, and in that case we could use the unsigned version.